dash/include/dash/matrix/MatrixRef.h

```diff
@@ -9,7 +9,9 @@
 #include <dash/Pattern.h>
 #include <dash/GlobRef.h>
 #include <dash/HView.h>
-#include <dash/Container.h>
+
+#include <dash/view/Local.h>
```

Comment:
Unfortuately there are many many cross dependencies and the easiest way was to separate the includes to specific files. And yes, this is required.